ui/src/main/js/view/remove-sidepanel.js

```diff
@@ -27,4 +27,5 @@ var jqProxy = require('../jQuery');
 exports.render = function (message, onElement) {
     var $ = jqProxy.getJQuery();
     $('div#side-panel').remove();
+    $('div#main-panel').addClass("fullscreen");
```

Comment:
@recena We're already removing the side-panel in this fashion.  Do you have a better way?  This was the approach that I was able to get working after several attempts, and is quite straight-forward.